exercises/etl/canonical-data.json

```diff
@@ -1,77 +1,76 @@
 {
   "exercise": "etl",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This is not the correct next version. Changing the input like this is not backwards compatible, so should probably be "2.0.0".